src/overlays/gamestates/ovl_select/z_select.c

```diff
@@ -43,144 +43,143 @@ void Select_LoadGame(SelectContext* this, s32 entranceIndex) {
+    { "28:\x8Dﾄﾋﾞｺﾐ ﾖｳｾｲ ｱﾅ", Select_LoadGame, 0x036D }, // "28: Fairy Dive Hole" ("28: Fairy's Fountain (Healing Fairies)")
+    { "29:\x8Dﾏﾎｳｾｷ ﾖｳｾｲﾉｲｽﾞﾐ", Select_LoadGame, 0x0371 }, // "29: Magic Stone Fairy's Fountain" ("29: Great Fairy's Fountain (Spells)")
+    { "30:\x8Cｶﾞﾉﾝ\x8Dｻｲｼｭｳｾﾝ", Select_LoadGame, 0x043F }, // "30: Final Battle With Ganon" ("30: Battle With Ganon")
+    { "31:\x8Cﾊｲﾗﾙ\x8Dﾅｶﾆﾜ", Select_LoadGame, 0x0400 }, // "31: Hyrule Inner Garden" ("31: Castle Courtyard")
+    { "32:\x8Dﾂﾘﾎﾞﾘ", Select_LoadGame, 0x045F }, // "32: Fishing" ("32: Fishing Pond")
+    { "33:\x8Cﾎﾞﾑﾁｭｳﾎﾞｰﾘﾝｸﾞ", Select_LoadGame, 0x0507 }, // "33: Bombchu Bowling" ("33: Bombchu Bowling Alley")
+    { "34:\x8Cﾛﾝﾛﾝ\x8Dﾎﾞｸｼﾞｮｳ ｿｳｺ 1", Select_LoadGame, 0x004F }, // "34: Lon Lon Ranch - Storehouse 1" ("34: Talon's House")
+    { "35:\x8Cﾛﾝﾛﾝ\x8Dﾎﾞｸｼﾞｮｳ ｿｳｺ 2", Select_LoadGame, 0x05D0 }, // "35: Lon Lon Ranch - Storehouse 2" ("35: Ranch Silo")
+    { "36:\x8Dﾐﾊﾘ ｺﾞﾔ", Select_LoadGame, 0x007E }, // "36: Lookout Hut" ("36: Guard House")
+    { "37:\x8Dﾏﾎｳ ﾉ ｸｽﾘﾔ", Select_LoadGame, 0x0072 }, // "37: Magic Potion Shop" ("37: Granny's Potion Shop")
+    { "38:\x8Dﾀｶﾗﾊﾞｺﾔ", Select_LoadGame, 0x0063 }, // "38: Treasure Chest Shop" ("38: Treasure Chest Game")
+    { "39:\x8Dｷﾝ \x8Cｽﾀﾙﾁｭﾗ ﾊｳｽ", Select_LoadGame, 0x0550 }, // "39: Gold Skulltula House" ("39: House of Skulltula")
+    { "40:\x8Dｼﾞｮｳｶﾏﾁ ｲﾘｸﾞﾁ", Select_LoadGame, 0x0033 }, // "40: Castle Town - Entrance" ("40: Market Entrance")
+    { "41:\x8Dｼﾞｮｳｶﾏﾁ", Select_LoadGame, 0x00B1 }, // "41: Castle Town" ("41: Market")
+    { "42:\x8Dｳﾗﾛｼﾞ", Select_LoadGame, 0x00AD }, // "42: Back Alley" ("42: Back Alley")
+    { "43:\x8Dﾄｷﾉｼﾝﾃﾞﾝ ﾏｴ", Select_LoadGame, 0x0171 }, // "43: In Front of the Temple of Time" ("43: Temple of Time Exterior")
+    { "44:\x8Dﾘﾝｸﾉｲｴ", Select_LoadGame, 0x00BB }, // "44: Link's House" ("44: Link's House")
+    { "45:\x8Cｶｶﾘｺ\x8Dﾑﾗﾉﾅｶﾞﾔ", Select_LoadGame, 0x02FD }, // "45: Kakariko Village Row House" ("45: Carpenter Boss's House")
+    { "46:\x8Dｳﾗﾛｼﾞﾉ ｲｴ", Select_LoadGame, 0x043B }, // "46: Back Alley House" ("46: Back Alley House (Man in Green)")
+    { "47:\x8Dｺｷﾘﾉﾑﾗ ﾓﾉｼﾘｷｮｳﾀﾞｲﾉｲｴ", Select_LoadGame, 0x00C9 }, // "47: Kokiri Village - Know-It-All Brothers' House" ("47: Know-It-All Brothers' House")
+    { "48:\x8Dｺｷﾘﾉﾑﾗ ﾌﾀｺﾞﾉｲｴ", Select_LoadGame, 0x009C }, // "48: Kokiri Village - Twins' House" ("48: Twins' House")
+    { "49:\x8Dｺｷﾘﾉﾑﾗ \x8Cﾐﾄﾞ\x8Dﾉｲｴ", Select_LoadGame, 0x0433 }, // "49: Kokiri Village - Mido's House" ("49: Mido's House")
+    { "50:\x8Dｺｷﾘﾉﾑﾗ \x8Cｻﾘｱ\x8Dﾉｲｴ", Select_LoadGame, 0x0437 }, // "50: Kokiri Village - Saria's House" ("50: Saria's House")
+    { "51:\x8Dｳﾏｺﾞﾔ", Select_LoadGame, 0x02F9 }, // "51: Stable" ("51: Stable")
+    { "52:\x8Dﾊｶﾓﾘﾉｲｴ", Select_LoadGame, 0x030D }, // "52: Grave Keeper's House" ("52: Gravekeeper's Hut")
+    { "53:\x8Dｳﾗﾛｼﾞ ｲﾇｵﾊﾞｻﾝﾉｲｴ", Select_LoadGame, 0x0398 }, // "53: Back Alley - Dog Lady's House" ("53: Back Alley House (Mamamu Yan)")
```

Comment:
Mamamu Yan is apparently the name this character got in mm. Would not include it in this game since afaik she is unnamed
Today i learned she had a name in mm though lol